### DISCUSSION TITLE
Can't find .env Imichh Dockage

### DISCUSSION THREAD

**Discussion by nreeb** [tteck/Proxmox#3573] [Help Needed] [answered]
I installed Imichh on Dockage and I can't find the .env to change my storage to my NAS

**Comment by clemone210** [comment]
the .env is located in `/opt/dockage/immich`
you can view it with `ls -a` within the directory
you can edit it with `nano .env` within the directory

**Comment by nreeb** [comment, marked as answer]
Thanks that got me on the right track.

It was in opt/stacks/immich

Appreciate it!
